boilerplate/.gitignore

```diff
@@ -1,2 +0,0 @@
-dist
```

Comment:
Are this file and the `playground/.gitignore` both removed to favor the root `.gitignore`?

Reply:
This was the plan but moving this to different PR as it's unrelated. 🙂